Support cloud storage sink (#10998)
pingcap/docs: ticdc/ticdc-changefeed-config.md

# Filter rules.
# Filter syntax: <https://docs.pingcap.com/tidb/stable/table-filter#syntax>.

> rules = ['*.*When the downstream is a storage service, the protocol can only be set to canal-json or csv.', '!test.*']

Reviewer: Is this a typo?

Author: Fixed in [68fb1b4](https://github.com/pingcap/docs/pull/10998/commits/68fb1b4b4fa1b6d9d27d1b9c0250d3d5a3b5eefc)

> rules = ['*.*When the downstream is a storage service, the protocol can only be set to canal-json or csv.', '!test.*']

Author: ```suggestion
rules = ['*.*', '!test.*']
```

# The following three configuration items are only used when you replicate data to storage sinks. You can ignore them when replicating data to MQ and MySQL sinks.
# Row terminator, used for separating two data change events. The value is empty by default, which means "\r\n" is used.
terminator = ''
# Date separator type of the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For details, see <https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>.
date-separator = 'none'
# Whether to use partitions as the separation string. The default value is false, which means that partitions in a table are not stored in separate directories. For details, see <https://docs.pingcap.com/zh/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records)>
enable-partition-separator = false

# Since v6.5.0, TiCDC supports saving data changes to storage services in CSV format. Ignore the following configurations if you replicate data to MQ and MySQL sinks.
[sink.csv]
# Delimiter between fields. The value must be an ASCII character and defaults to `,`.
delimiter = ','

> # The quotation that surrounds the fields. The default value is `"`, which means no quotation is used.

Author: ```suggestion
# The quotation that surrounds the fields. The default value is `"`. When the value is empty, no quotation is used.
```

> quote = '"'

Reviewer: This is incorrect. This translation is not consistent with the Chinese version. The default value `"` means double quote will be used.

Author: Fixed in [68fb1b4](https://github.com/pingcap/docs/pull/10998/commits/68fb1b4b4fa1b6d9d27d1b9c0250d3d5a3b5eefc)
